### ISSUE TITLE
aria-multiselectable should be on accordions container

### ISSUE THREAD

**Issue by philbuchanan** [philbuchanan/Accordion-Shortcodes#64]
It is currently set on each individual accordion item title.

Reference: https://v4-alpha.getbootstrap.com/components/collapse/
